web/controllers/dashboard_controller.ex

```diff
@@ -55,17 +55,16 @@ defmodule HexWeb.DashboardController do
 
   def add_email(conn, params) do
     user = Users.with_emails(conn.assigns.logged_in)
-
+    email = params["email"]["email"]
     case Users.add_email(user, params["email"], audit: audit_data(conn)) do
       {:ok, _user} ->
-        email = params["email"]["email"]
         conn
         |> put_flash(:info, "A verification email has been sent to #{email}.")
         |> redirect(to: dashboard_path(conn, :email))
-      {:error, changeset} ->
+      {:error, _changeset} ->
         conn
-        |> put_status(400)
-        |> render_email(user, changeset)
+        |> put_flash(:error, email_error_message(:exists, email))
```

Comment:
@ericmj My thought was that rather than giving the changeset errors which might contain potentially other details that we just might wanna show up to users, we would just give this error on `add email`. Also, in most cases, I thought it would be duplicate e-mails that would cause to hit the error. Thoughts?